.editorconfig

```diff
@@ -34,7 +34,7 @@ dotnet_style_qualification_for_event                       = false : suggestion
 
 # Use language keywords instead of framework type names for type references
 dotnet_style_predefined_type_for_locals_parameters_members = true : suggestion
-dotnet_style_predefined_type_for_member_access             = true : suggestion
+dotnet_style_predefined_type_for_member_access             = false : suggestion
```

Comment:
Why did you turn this false?

Reply:
Because VS was giving me squigglies for a lot of existing code, and it seemed type name usage was in majority, so I changed to the value that fixed the most issues. I did it happily. :) Why, have we adopted a different rule now than in O1?